data/campaigns/tutorial/lua/character_selection.lua

```diff
@@ -11,7 +11,10 @@ function wml_actions.select_character()
 	local character_selection_dialog = wml.load "campaigns/tutorial/gui/character_selection.cfg"
 	local dialog_wml = wml.get_child(character_selection_dialog, 'resolution')
 
-	local character = gui.show_dialog(dialog_wml)
+	local character = wesnoth.sync.evaluate_single(function()
+		local result = false
```

Comment:
This `result` is unused., and there's odd whitespace on the next line.

However, as I've just posted in #5994, it makes sense for whichever of us is going to do SotA to do the tutorial too.